conventional_pre_commit/format.py

```diff
@@ -79,7 +92,7 @@ def conventional_types(types=[]):
     return types
 
 
-def is_conventional(input, types=DEFAULT_TYPES, optional_scope=True):
+def is_conventional(input, types=DEFAULT_TYPES, optional_scope=True, scopes: Optional[list[str]] = None):
```

Comment:
This type hint should use the `typing.List` to avoid an error in Python 3.8

Reply:
Resolved with commit f51f6621b3d4d0ca1cca5d226ed8d57bc2c032de.